cmd/plume/prerelease.go

```diff
@@ -641,8 +793,10 @@ func awsPreRelease(ctx context.Context, client *http.Client, src *storage.Bucket
 		}
 	}
 
-	if err := awsUploadAmiLists(ctx, src, spec, &amis); err != nil {
-		return fmt.Errorf("uploading AMI IDs: %v", err)
+	if selectedSystem == "cl" {
+		if err := awsUploadAmiLists(ctx, src, spec, &amis); err != nil {
```

Comment:
Do the Fedora AMI IDs not need to be stored anywhere?

Reply:
@dustymabe Do we need to store this?

Reply:
This is something we are not doing right now. In the past, we had the necessity of the list of ami ids to delete the older ones periodically but I had a different approach there. I wanted to distinguish the AMIs using the AWS tags.

Reply:
We are essentially using fedmsg right now to retrieve AMI IDs. As long as we are still sending out fedmsg I would say we're fine not storing them otherwise and we can improve in the future.

Reply:
@sayanchowdhury - can you confirm we'll still be sending out fedmsg?

Reply:
Yes, we will be sending out the fedmsg messages.

Reply:
Out of curiousity how are you getting the AMIs from to send out with `fedmsg`?